src/pages/filter/ui/attribute-header.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactElement } from 'react'
+import cn from 'classnames'
+import Tooltip from 'rc-tooltip'
+
+import { Icon } from '@ui/icon'
+import { FnLabel } from '@components/fn-label'
+import { AttributeKinds, TPropertyStatus } from '@service-providers/common'
+
+interface IAttributeHeader {
+  className?: string
+  attrStatus: TPropertyStatus
+}
+
+// TODO: variants counts (selected/all) from new design
+export const AttributeHeader = ({
+  attrStatus,
+  className,
+}: IAttributeHeader): ReactElement => {
```

Comment:
no need to specify return type

Reply:
i think, that all exported functions (and public members of classes)  should have a return type. Because when i see the definition of function that i called, i want to see the return type of it